integration-tests/src/test/java/org/zowe/apiml/functional/gateway/ValidateAPITest.java

```diff
@@ -86,6 +87,19 @@ void testGetEndpointNonConformant() {
 
     }
 
+    @Test
+    @TestsNotMeantForZowe
+    void testGetEndpointWithNoAuthentication() {
```

Comment:
Is it only one endpoint to verify that is now protected with authentication?

Reply:
I can add it for the other endpoint too